po/plume/hr.po

```diff
@@ -65,10 +73,56 @@ msgstr ""
 msgid "You can't use this media as a blog banner."
 msgstr ""
 
+# src/routes/blogs.rs:312
+msgid "Your blog information have been updated."
+msgstr ""
+
+# src/routes/comments.rs:89
+msgid "Your comment have been posted."
+msgstr ""
+
+# src/routes/comments.rs:163
+msgid "Your comment have been deleted."
+msgstr ""
+
+# src/routes/instance.rs:145
+msgid "Instance settings have been saved."
+msgstr ""
+
+# src/routes/instance.rs:182
+msgid "{} have been unblocked."
+msgstr ""
+
+# src/routes/instance.rs:184
+msgid "{} have been blocked."
+msgstr ""
+
+# src/routes/instance.rs:218
+msgid "{} have been banned."
+msgstr ""
+
 # src/routes/likes.rs:51
 msgid "To like a post, you need to be logged in"
 msgstr ""
 
+# src/routes/medias.rs:143
+msgid "Your media have been deleted."
+msgstr ""
+
+# src/routes/posts.rs:140
+#, fuzzy
+msgid "You are not allowed to delete this media."
+msgstr "Ti ne autor ovog bloga."
```

Comment:
this is supposed to be

> Ti nisi autor ovog(a?) bloga

but i just realised that fuzzy means it was auto translated…

Reply:
Yes, it was auto-translated. Once it will be on Crowdin, you will be able to fix it. :wink: